src/apps/weblib/ts-api/events.ts

```diff
@@ -124,6 +124,8 @@ export interface Caption extends TextElement {
 export interface Legend extends Element {
 	tagName: 'legend'
 	channel: string
+	/** Marker labels y overflow by pixel */
+	yOverflow: number
```

Comment:
Would be good to have a HTML element compatible parameter here, since the events and their elements are mimicking DOM events and Elements. 
There is no analogous parameter in DOM elements, the closest which used for the same purpose: 
https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTop
https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollHeight

Reply:
I'll rename `yOverflow` to `scrollTop`.

Reply:
So, the style parameters are named based on CSS attributes, and Element properties named based on HTML element property names.
The equivalent would be in JS:
```
LegendElement.style.translateY = xxx
console.log(LegendElement.scrollTop)
```
So translateY is ok, because it is CSS like enough. But in the event element we need HTMLElement-like names.

Also scrollTopMax is non standard and it does not give enough information, since it does not represent the actual position, but the max position.

Reply:
The actual position is represented by the style parameter which will be redundant if the event contains too

Reply:
The style parameter is showing the target position, not the actual position during animation.